client/src/components/BookingCard/BookingCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Typography } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import { Box } from '@mui/material';
+import { Booking } from '../../interface/Booking';
+
+interface Props {
+  booking: Booking;
+  isNextBooking?: boolean;
+}
+
+export default function BookingCard({ booking, isNextBooking }: Props): JSX.Element {
+  const date = () => {
```

Comment:
Just a note: These type of functions can be moved to a `utils` folder, in case we need to use the same date format in other components. However, for our project I don't think this is the case, so no action needed 😄